Formula/fstar.rb

```diff
@@ -37,16 +38,20 @@ def install
 
     system "opam", "config", "exec", "--", "make", "-C", "src", "boot-ocaml"
 
-    bin.install "bin/fstar.exe"
+    bin.install "bin/fstar.exe" => "_fstar.exe"
```

Comment:
Sorry, I misunderstood your first comment.
I updated the formula to put the `fstar.exe` executable in `libexec/bin/fstar.exe` and the wrapper script in `bin/fstar.exe`.